rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy

```diff
@@ -2916,22 +2915,28 @@ Format is a string like `2d1h4n5s` using the following characters for time units
                     && extra.futureScheduledExecutions.size() > max) {
                 extra.futureScheduledExecutions = extra.futureScheduledExecutions[0..<max]
             }
-        } else if(request.api_version >= ApiVersions.V48) {
-            def map = scheduledExecutionService.nextOneTimeScheduledExecutions([scheduledExecution] as List<ScheduledExecution>)
+        }
+
+        def map = scheduledExecutionService.nextOneTimeScheduledExecutions([scheduledExecution] as List<ScheduledExecution>)
```

Comment:
this still needs a tweak, the nextOneTimeScheduledExecutions should only be returned if appropriate API version is used